### DISCUSSION TITLE
Exporting Notebooks from the REPL or Shell

### DISCUSSION THREAD

**Discussion by Nagefire** [fonsp/Pluto.jl#939] [Questions about using Pluto] [answered]
I use Pluto to take class notes that are stored in a private git repository. I have CI/CD set up for my repository, and I would like to publish static HTML pages for each notebook every time I update them, but I can't figure out how to export the notebooks outside of opening the Pluto page and clicking the link. Is there a way to open a notebook and export it as HTML from the REPL or bash?

**Comment by fonsp** [comment, marked as answer]
Hey! We are currently working on this functionality, check back later!
